src/test/java/testPages/HomePageTest.java

```diff
@@ -22,4 +23,20 @@ public void verifySmartphonePageTitle() {
 
         Assert.assertEquals(actualSmartphonePageTittle, "Мобильные телефоны", "The actual page title doesn't match expected");
     }
+
+    @Test(priority = 4, description = "[TC#4] -  Check logo on the site")
+    @Description("Test case description:  Check logo on the site")
+    public void verifyLogoOnTheSite() {
+        DriverManager.getDriver().navigate().refresh();
+        String currentURL = new HomePage().clickOnTheLogotype().getPageUrl();
+        Assert.assertEquals(currentURL, ONLINER_URL, "The actual page doesn't match expected");
+    }
+
+    @Test(priority = 3, description = "[TC#3 - Check searching system")
+    @Description("Test case description: check searching system with negative data")
+    public void verifySearchingSystemWithNegativeData() {
+        String actualSearchData = new HomePage().textInput().getSearchData();
```

Comment:
what does mean textInput() method? that's not clear
I don't see where and WHAT you insert in the search field.